test/inspector/test-async-stack-traces-promise-then.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+const common = require('../common');
+common.skipIfInspectorDisabled();
+const helper = require('./inspector-helper');
+const assert = require('assert');
+
+const script = 'Promise.resolve().then(() => { debugger; });';
```

Comment:
Inside of V8 we have own promise instrumentation, which is partially based on V8 async hooks, I'm not sure how these two instrumentations will live together, can we check msg.params.asyncStackTrace content in this test?